src/mpid/ch4/netmod/ucx/ucx_rma.h

```diff
@@ -298,6 +298,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_UCX_do_get(void *origin_addr,
                                         target_contig, target_bytes, target_true_lb);
     MPIDI_Datatype_check_contig_size_lb(origin_datatype, origin_count,
                                         origin_contig, origin_bytes, origin_true_lb);
+    MPIR_ERR_CHKANDJUMP((origin_count != target_count), mpi_errno, MPI_ERR_SIZE, "**rmasize");
 
```

Comment:
This check is incorrect. The counts might be different if `origin_datatype != target_datatype ` . You should check `bytes` instead of `count`.

Reply:
You are right. This is an oversight. Fixed.